### DISCUSSION TITLE
How to generate full length photo?

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2946] [Q&A] [answered]
I tried the prompt as: full length Beautiful girl, smiling, New York City, spring, blooming flowers, Central Park, iconic skyline, Empire State, bustling fashionable beautiful people, cars, streets, cabs, and Chrysler Buildings
image size: 768*1344
But the generated images are usually close-up.

I don't know how to do make a prompt or configure the app for perfect full length.

Please let me know if you have any ideas on it. Thank you.

![2024-05-18_03-09-41_2202](https://github.com/lllyasviel/Fooocus/assets/91205658/061df8be-5dd6-4bfb-86c9-72ae5e8dad18)

This is the image from the Turbo Speed. The performance is good but the quality is so bad as you can see.
I would like to know how to improve the quality as well as the background details.

Thank s

**Comment by bsdesigner** [comment, marked as answer]
https://bsdesigners.wixstudio.io/shot-cam

**Comment by xhoxye** [comment]
Turn off all styles and use "Full Length Portrait Photoshoot" at the beginning of the prompt.Turbo is not used.
![QQ截图20240519100500](https://github.com/lllyasviel/Fooocus/assets/129571231/42994a2d-05df-4e2f-b646-e31cd2698ff6)
![QQ截图20240519100701](https://github.com/lllyasviel/Fooocus/assets/129571231/881d2142-286c-4a1c-b161-86add64540b7)
